src/include/columnar/columnar_storage.h

```diff
@@ -17,6 +17,14 @@
 #include "storage/smgr.h"
 #include "utils/rel.h"
 
+#include "columnar/columnar_tableam.h"
+
+
+#define COLUMNAR_INVALID_ROW_NUMBER 0
+#define COLUMNAR_FIRST_ROW_NUMBER 1
+#define COLUMNAR_MAX_ROW_NUMBER \
+	(COLUMNAR_FIRST_ROW_NUMBER + VALID_ITEMPOINTER_OFFSETS * VALID_BLOCKNUMBERS)
```

Comment:
I wanted to share the calculation that I just made around those macros:

given that:
```
COLUMNAR_FIRST_ROW_NUMBER = 1
VALID_ITEMPOINTER_OFFSETS = MaxOffsetNumber - FirstOffsetNumber = BLCKSZ / sizeof(ItemIdData) - FirstOffsetNumber = 8192 / 4 - 1 = 2047
VALID_BLOCKNUMBERS = MaxBlockNumber + 1 = 0xFFFFFFFE + 1 = 4294967295
```

`COLUMNAR_MAX_ROW_NUMBER` evaluates to:
```
COLUMNAR_MAX_ROW_NUMBER = (COLUMNAR_FIRST_ROW_NUMBER + VALID_ITEMPOINTER_OFFSETS * VALID_BLOCKNUMBERS)
COLUMNAR_MAX_ROW_NUMBER = 1 + 2047 * 4294967295 = 8791798052866
```